app/livechat/server/api/v1/room.js

```diff
@@ -6,7 +6,7 @@ import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
 import { settings as rcSettings } from '../../../../settings';
 import { Messages, LivechatRooms } from '../../../../models';
 import { API } from '../../../../api/server';
-import { findGuest, findRoom, getRoom, settings, findAgent, onCheckRoomParams } from '../lib/livechat';
+import { findGuest, findRoom, getRoom, settings, findAgent, onCheckRoomParams, findOpenRoom, findDepartment, getRoomById, updateGuest } from '../lib/livechat';
```

Comment:
IMO the logic introduced here in the endpoint scope should be placed inside `getRoom` method ather than here. `getRoom` method is imported in other places too, so logic must be reused. 
Therefore, the new methods stated here would not be needed.